react-search-component/my-app/src/SearchableList.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+import './App.css';
+
+export default function SearchableList({ list }) {
+  return (
+    <div>
+      <Input item={list} />
+    </div>
+  );
+}
+
+function Input({ item }) {
```

Comment:
It's odd to have this named `Input` when it both handles the input and the results.